crates/symbolicator/src/services/symbolication.rs

```diff
@@ -420,10 +427,18 @@ impl SymbolicationActor {
         // Assume that there are no UUID4 collisions in practice.
         let requests = self.requests.clone();
 
-        if let Ok(num_requests) = requests.lock().len().try_into() {
+        let num_requests = requests.lock().len();
+        if let Ok(num_requests) = num_requests.try_into() {
             metric!(gauge("requests.in_flight") = num_requests);
         }
```

Comment:
this `try_into` is converting a usize to u64, right? We have a bunch of `as u64` in various places already, so might as well use that.